docs/tutorial.rst

```diff
@@ -498,7 +498,7 @@ the values used.
 .. todo:: Add a diagram of the model for convenience.
 
 .. code-block:: python
-
+    import math
```

Comment:
We need a blank line under the ``code-block`` declaration @koelling, otherwise the docs don't build.